src/main/resources/application.yml

```diff
@@ -21,6 +21,7 @@ spring:
     store-type: jdbc
     jdbc:
       initialize-schema: never
+    timeout: 20M
```

Comment:
So the default for session timeout is 30 mins.  I have reduced this to 20 mins on the back of the BSI recommendation.